lib/rspec/core/runner.rb

```diff
@@ -110,6 +110,9 @@ def setup(err, out)
       def run_specs(example_groups)
         @configuration.reporter.report(@world.example_count(example_groups)) do |reporter|
           @configuration.with_suite_hooks do
+            if @world.example_count(example_groups) == 0 && @configuration.fail_if_no_examples
+              return 1
```

Comment:
I suspect that the build is going to fail due to this line not being covered by any specs, but if you follow my advice above about adding specs for this it should solve that issue.